libstuff/libstuff.cpp

```diff
@@ -2452,22 +2452,29 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     uint64_t elapsed = STimeNow() - startTime;
 
     // Warn if it took longer than the specified threshold
-    if ((int64_t)elapsed > warnThreshold)
-        SWARN("Slow query (" << elapsed / 1000 << "ms) :" << sql);
+    string sqlToLog = sql;
+    if ((int64_t)elapsed > warnThreshold) {
+        string match;
+        const bool hasAuthToken = SREMatch(".*(\"authToken\"\\:\"[0-9A-F]{400,1024}\").*", sql, match);
```

Comment:
Should we just match `[0-9A-F]{400,1024}` instead ? Probably not necessary ?

Reply:
Sorry, what's the difference with what's here? Remove the `authToken:` part?

Reply:
yes